python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -264,13 +264,18 @@ class ReferenceObjectLoader:
         An iterable object of data IDs that point to reference catalogs.
     refCats : iterable of `lsst.daf.butler.DeferredDatasetHandle`
         Handles to load refCats on demand.
+    name : `str`
```

Comment:
Should add `, optional`

Reply:
I'd rather we treated it as defacto required, since knowing what a refobject loader actually has loaded is very useful, but I guess we probably can't make it "silently optional"?